contrib/coreos/cloud-config

```diff
@@ -0,0 +1,266 @@
+#cloud-config
```

Comment:
Would you mind removing this file or moving it somewhere else if this is meant to be the static cloud config to be uploaded to S3?